src/pages/AddHotel/AddHotelForm.jsx

```diff
@@ -0,0 +1,75 @@
+/* eslint linebreak-style: ["error", "windows"] */
+// import { IKContext, IKUpload } from "imagekitio-react";
+import React, { useRef, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form';
+import { AddHotel } from '../../redux/hotels/hotels';
+// import { AddHotel } from '../../redux/hotels/hotels';
+// import { AddHotel } from '../../redux/hotels/hotels';
```

Comment:
- [ ] Kindly remove this old piece of commented code to improve the repo's cleanliness. 👍